blog-cse/2023-04-28-content.md

```diff
@@ -0,0 +1,70 @@
+* [New] THRESHOLD-S00114 HTTP Response Error Spike to AWS EKS; HTTP web services provide response codes to client requests. The response code numbers in the 400s are used to indicate a client related error and response code numbers in the 500s represent server related errors. This rule looks for a AWS EKS cluster receiving a large frequency of web errors within a short period of time. It is unusual for a web client to cause this many errors in a short period of time. Common occurrences for this behavior is scanning/probing activity or scripted web clients which are now encountering errors due to a misconfiguration or recent change. This rule alerts when a host on the monitored network triggers the threshold.
+* [New] MATCH-S00868 New Binding Role Created on AWS EKS; A role binding grants a resource superuser or administrative access to a Kubernetes cluster. Ensure this action is expected and performed by known Kubernetes administrators.
+* [New] MATCH-S00867 New Cluster Admin Binding Role Created on AWS EKS; A cluster-admin role binding grants a resource superuser or administrative access to a Kubernetes cluster. Ensure this action is expected and performed by known Kubernetes administrators.
+* [New] MATCH-S00866 Privileged Pod Created on AWS EKS; Privileged containers have all capabilities of the host machine. These privileged containers may perform actions directly on the host that they are running on. Ensure that this event is expected and occurs from a user account or IP address that normally works with privileged containers within the cluster. Customers are encouraged to set up an exclusion list for spec.securitycontext.capabilities for pods that are frequently going to be managed with privileged escalation.
+* [Updated] MATCH-S00864 Azure Firewall Rule Modified
+* [Updated] MATCH-S00839 Azure Virtual Machine RunCommand Issued
+* [Updated] FIRST-S00021 First Seen Azure Virtual Machine Run Command Issued by User
+* [Updated] FIRST-S00032 First Seen Kubectl Command From User; Expanded record types considered to include Endpoint.
+* [Updated] CHAIN-S00012 Potential Azure Persistence via Automation Accounts
+* [Updated] MATCH-S00167 Recon Using Common Windows Commands; Narrowed rule criteria to Windows executables to prevent erroneous matches from *nix based systems
+
+#### Log Mappers
+Adds support for Druva events via C2C. Modifies Darktrace mappers to include alternate field values and supports additional events. Modified CyberArk EPM mappers to include alternate field values. Adds support for DocuSign Monitor events via C2C. Adds support for Cisco Meraki events ingested via C2C. Includes updates to Microsoft Azure normalization to get more granular data for actions being performed. Adds additional mapping support for Jamf events. Corrects bug in severity mapping for Sysdig mappers. Expands mapping support for Workday logs ingested via C2C.
```

Comment:
Are you sure you want this all as one paragraph? You might just want to add these pieces to lines below. For example:
"[New] Druva inSync - Catch All: Adds support for Druva events via C2C."

Reply:
We're trying out some new things and continuing to iterate on summaries versus in line comments. Would it make sense for trying it both ways and soliciting feedback?

Reply:
@ptobia, I'd go one way or the other, but not both in the same set of release notes.